pkg/pillar/cmd/volumemgr/handlework.go

```diff
@@ -248,6 +253,13 @@ func casIngestWorker(ctxPtr interface{}, w worker.Work) worker.WorkResult {
 	if err != nil {
 		result.Error = err
 		result.ErrorTime = time.Now()
+	} else {
+		if ctx.hvTypeKube {
+			cfg := lookupContentTreeConfig(ctx, status.ContentID.String())
+			if cfg != nil && status.IsContainer() {
+				publishContentTreeStatus(ctx, &status)
+			}
```

Comment:
Why do you need this code? Are you missing a publish somewhere else?

Reply:
Or is this leftover code from debugging something else?

Reply:
Still don't have an answer to this question.

Reply:
Actually not needed I guess. This code was written many months ago to get the native container part working, I do not see any reason to publish content tree status from here. I will clean up.